_config.yml

```diff
@@ -16,6 +16,7 @@ exclude: ["_scripts", "tests", "apidocs/layouts", "Gemfile", "hooks", "index.htm
 # Google Analytics, etc.
 # TODO these should only be set when building for production, or passed as an environment variable when building
 google_analytics: GTM-WL2QLG5
+polldaddy_id: 8453675
```

Comment:
If we don't set these values, the analytics scripts and polldaddy will not be included. We should look if we can disable these when building the (netlify) previews, and when building locally. Not urgent, but something we could look at.

Reply:
Agree. We don't need the tracking script for Netlify previews and local builds.